docs/source/common/optimizers.rst

```diff
@@ -252,6 +252,29 @@ If you want to call schedulers that require a metric value after each epoch, con
 
 -----
 
+Bring your own Custom Learning Rate Schedulers
```

Comment:
This is ordered under "Manual Optimization", probably not intentional.

Reply:
the structure of this doc is still not appropriate. I am reviewing this one in another PR. Will be restructuring it there.